pillar/base/haproxy.sls

```diff
@@ -155,3 +171,16 @@ haproxy:
       extra:
         - timeout client 86400
         - timeout server 86400
+
+    {% for (port, service, ssl) in [(25, "smtp", False), (587, "smtps", True), (465, "submission", True)] %}
+    roundup-{{ service }}:
+      bind: :{{ port }}
+      service: roundup-{{ service }}
+      extra:
+        - timeout client 30m
+        - timeout server 30m
+        {% if ssl %}
+        {# TODO: Need to update pillar data and haproxy.cfg.jinja to utilize tls #}
```

Comment:
This needs implementing still

https://www.haproxy.com/documentation/haproxy-configuration-tutorials/ssl-tls/ says it needs a bind port

We already do 
```
{% for item in config.get("extra", []) -%}
{{ item }}
{% endfor -%}
```
So we will need special casing or need to move it out of the `extra` tree (?)

Reply:
[5d93dbc](https://github.com/python/psf-salt/pull/393/commits/5d93dbc1a618575a16c8f0fdc5ba7d82bee22a63) aims to address this @ewdurbin